Upgrade to 4.0.3 failes - ioBroker is unable to start

````
pi@raspberrypi:/opt/iobroker $ iob upgrade self
Update js-controller from @3.3.22 to @4.0.3
NPM version: 6.14.15
npm install iobroker.js-controller@4.0.3 --loglevel error --unsafe-perm --prefix "/opt/iobroker" (System call)
In file included from ../../nan/nan.h:56,
                 from ../src/unix_dgram.cc:5:
/home/iobroker/.cache/node-gyp/14.18.2/include/node/node.h:787:43: warning: cast between incompatible function types from ‘void (*)(v8::Local<v8::Object>)’ to ‘node::addon_register_func’ {aka ‘void (*)(v8::Local<v8::Object>, v8::Local<v8::Value>, void*)’} [-Wcast-function-type]
       (node::addon_register_func) (regfunc),                          \
                                           ^
/home/iobroker/.cache/node-gyp/14.18.2/include/node/node.h:821:3: note: in expansion of macro ‘NODE_MODULE_X’
   NODE_MODULE_X(modname, regfunc, NULL, 0)  // NOLINT (readability/null_usage)
   ^~~~~~~~~~~~~
../src/unix_dgram.cc:404:1: note: in expansion of macro ‘NODE_MODULE’
 NODE_MODULE(unix_dgram, Initialize)
 ^~~~~~~~~~~
Server Objects 127.0.0.1:38042 Error from InMemDB: Error: GET-UNSUPPORTED for namespace cfg.: Data=["meta.objects.features.useSets"]
Server States 127.0.0.1:34156 Error from InMemDB: Error: GET-UNSUPPORTED for namespace meta.: Data=["meta.states.protocolVersion"]
Server Objects 127.0.0.1:38042 Error from InMemDB: Error: GET-UNSUPPORTED for namespace cfg.: Data=["meta.objects.protocolVersion"]
Server States 127.0.0.1:34158 Error from InMemDB: Error: PSUBSCRIBE-UNSUPPORTED for namespace meta.: Data=["meta.*"]
Server Objects 127.0.0.1:38042 Error from InMemDB: Error: Unknown LUA script load
npm ERR! code ELIFECYCLE
npm ERR! errno 22
npm ERR! iobroker.js-controller@4.0.3 install: `node iobroker.js setup first`
npm ERR! Exit status 22
npm ERR! 
npm ERR! Failed at the iobroker.js-controller@4.0.3 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/iobroker/.npm/_logs/2022-02-05T21_29_40_678Z-debug.log
host.raspberrypi Cannot install iobroker.js-controller@4.0.3: 22
````

````
pi@raspberrypi:/opt/iobroker $ node -v
v14.18.2
````

Do you need  /home/iobroker/.npm/_logs/2022-02-05T21_29_40_678Z-debug.log as well?

Now ioBroker does not start again:
````
iob update
internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module '/opt/iobroker/node_modules/iobroker.js-controller/iobroker.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
pi@raspberrypi:/opt/iobroker $ iob upgrade self
internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module '/opt/iobroker/node_modules/iobroker.js-controller/iobroker.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
pi@raspberrypi:/opt/iobroker $ iob start
pi@raspberrypi:/opt/iobroker $ 
````

iob start throws no error - but ioBroker is not reachable.

--- a/packages/db-objects-redis/lib/objects/objectsInRedisClient.js
+++ b/packages/db-objects-redis/lib/objects/objectsInRedisClient.js
@@ -4413,7 +4413,9 @@ class ObjectsInRedisClient {
                 } catch (e) {
                     script.loaded = false;
                     this.log.error(this.namespace + ' Cannot load "' + script.name + '": ' + e.message);
-                    throw new Error(`Cannot load "${script.name}" into objects database: ${e.message}`);
+                    if (!script.name.startsWith('redlock_')) {
+                        throw new Error(`Cannot load "${script.name}" into objects database: ${e.message}`);
+                    }
                 }
                 script.hash = hash;
             }

